test/fake_cloud_firestore_test.dart

```diff
@@ -534,6 +534,16 @@ void main() {
   });
 
   group('FieldValue', () {
+    test('Equality', () {
```

Comment:
Would you mind adding the permalink to the source tests? This way we can easily adapt if they change their tests in the future.

Reply:
Sure, will do.